prbt_moveit_config/launch/moveit_planning_execution.launch

```diff
@@ -33,10 +33,10 @@ limitations under the License.
   <!-- Set gripper -->
   <arg name="gripper" default="" />
 
-  <!-- Set STO handler -->
-  <arg name="sto" default="pnoz"/>
-  <arg name="read_api_spec_file" default="$(find prbt_hardware_support)/config/modbus_read_api_spec_$(arg sto).yaml" />
-  <arg name="write_api_spec_file" default="$(find prbt_hardware_support)/config/modbus_write_api_spec_$(arg sto).yaml" />
+  <!-- Set safety HW handler -->
+  <arg name="safety_hw" default="pnoz"/>
```

Comment:
What's the default? On melodic it is PSS4000, here it is PNozMulti? Adapt other parameters like `run_permitted_modbus_server_ip`, etc. accordingly, if PSS4000 is the default.

Reply:
In general: The default parameter `has_operation_mode_support` is not set as discussed. Please check.

If PSS4000 is the default safety hardware, please also adapt default value for parameter `has_braketest_support`.

Reply:
Is it maybe possible to store the parameter like `has_operation_mode_support`, etc. configuration for each safety hardware in a separate launch file and simply load it, if the corresponding safety hardware is selected? This would allow the reader, to simply overview the default configuration for each safety hardware.

Reply:
For this file, the default is sim=true. Therefore these settings would make sense.

Reply:
I think i solved this now, too. See what you think